src/seedu/addressbook/commands/ViewAllCommand.java

```diff
@@ -37,4 +37,6 @@ public CommandResult execute() {
             return new CommandResult(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
         }
     }
+
+    public boolean isMutating() { return false; }
```

Comment:
Coding standard violation. Please refer to Java coding standard to format the method properly.